packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -104,6 +125,58 @@ TextStyle getLabelStyle(WidgetTester tester) {
   ).style;
 }
 
+Finder findCustomLabel() {
+  return find.byKey(customLabelKey);
+}
+
+Rect getCustomLabelRect(WidgetTester tester) {
+  return tester.getRect(findCustomLabel());
+}
+
+Offset getCustomLabelCenter(WidgetTester tester) {
+  return getCustomLabelRect(tester).center;
+}
+
+Finder findInputText() {
+  return find.text(inputText);
+}
+
+Rect getInputRect(WidgetTester tester) {
+  return tester.getRect(findInputText());
+}
+
+Offset getInputCenter(WidgetTester tester) {
+  return getInputRect(tester).center;
+}
+
+Finder findHint() {
+  return find.text(hintText);
+}
+
+Rect getHintRect(WidgetTester tester) {
+  return tester.getRect(findHint());
+}
+
+Offset getHintCenter(WidgetTester tester) {
+  return getHintRect(tester).center;
+}
+
+double getHintOpacity(WidgetTester tester) {
+  return getOpacity(tester, hintText);
+}
+
+Finder findDecorator() {
+  return find.byType(InputDecorator);
+}
+
+Rect getDecoratorRect(WidgetTester tester) {
+  return tester.getRect(findDecorator());
+}
+
+Offset getDecoratorCenter(WidgetTester tester) {
+  return getDecoratorRect(tester).center;
+}
```

Comment:
Thanks for all of these helper methods :+1: . It makes the tests read much cleaner too.